README.md

```diff
@@ -4,11 +4,73 @@ This repository contains the source code for SKDB and its associated
 components.
 
 
-## Dependencies
+## Running Prepackaged SKDB
 
-You will need at least the following installed:
+You can run the latest release of SKDB's development server with Docker:
 
- * LLVM 15.
- * wasm-ld 15 (this is part of LLVM, but your distribution might split it into
-   a separate package: your wasm-ld version must match your LLVM version).
- * typescript 5.2 or later.
+```sh
+docker run -it -p 3586:8080 skiplabs/skdb-dev-server
+```
+
+The three SKDB packages (`skdb`, `skdb-dev`, and `skdb-react`) are all
+available as normal NPM packages.
+
+
+## Building From Source
+
+It is recommended that you use the prepackaged version of SKBD in most
+situations.
+
+For those who need to build from source, you will need a number of dependencies
+installed: SKDB's
+[`Dockerfile`](https://github.com/SkipLabs/skdb/blob/main/Dockerfile) serves as
+a good guide to those dependencies, though you may need to adapt them for your
+operating system. Note that you will need LLVM 15, and that other tools
+relevant to LLVM need to be compatible with LLVM 15.
+
+If your distribution/OS doesn't include typescript, you can install your own
+with `npm install --prefix=/path/to/install typescript`.
+
+
+### Build steps
+
+These build steps can be simplified for your first build, but for simplicity's
+sake we assume that you are doing multiple builds, and need to clean artifacts
+from previous builds: this is also safe, if unnecessary, for your first build.
+
+First build the bootstrap compiler:
+
+```sh
+cd /path/to/skdbrepo/compiler && make clean && default STAGE=0
+```
+
+Then build the rest of the system ensuring that `$PATH` points to the compiler
+built in the previous stage:
+
+```sh
+cd /path/to/skdbrepo && make clean && PATH=/path/to/skdbrepo/compiler/stage0/bin:$PATH make
```

Comment:
Let's not encourage cleaning? I think it looks amateur and the PR aspect of our readme is important for the product success. Cleaning is not necessary for skdb. I think it might be for the compiler, but for somewhat good reasons?